### PR TITLE
Merge potree truth-annotation-tool into develop

### PR DESCRIPTION
**Pull request by vrajur-nd**
Merges truth-annotation-tool into develop (see #53)

Main Additions:
 - (Viz) Button for switching to lane annotation mode
 - (Viz) Button for downloading lanes
 - (Viz) RTK Trajectory added to sidebar as vehicle layer (hidden by default)
 - python script for converting downloaded lanes from json to flatbuffer

Instructions: [finishing-a-feature-branch](https://github.com/NextDroid/SoftwareDevelopment/wiki/Git-Workflow-Feature-Development#finishing-a-feature-branch)

### CODE REVIEW

File: data/.~lock.rtk.csv# [hunk 1]
```diff
@@ -0,0 +1 @@
+,vrajur,Vinay-Precision-7510,24.07.2019 13:08,file:///home/vrajur/.config/libreoffice/4;
```
Comment (bcasselsND) [inline data/.~lock.rtk.csv#]: This doesn't look like something you wanted to check in. Maybe tweak the .gitignore?

File: demo/radar.html [hunk 2]
```diff
@@ -639,13 +685,87 @@
...
+			viewer.scene.scene.remove(removeLanes);
+			removeLanes = viewer.scene.scene.getChildByName("Lanes");
+		}
+
+		// TOGGLE window.annotateLanesMode
+		window.annotateLanesMode = !window.annotateLanesMode;
+
+		// Disable Button:
+
+		debugger; // check s3, bucket, name
```
Comment (bcasselsND) [inline demo/radar.html]: Remove the `debugger`?

File: src/utils/Measure.js [hunk 3]
```diff
@@ -186,6 +186,33 @@ export class Measure extends THREE.Object3D {
...
+				if (window.truthAnnotationMode == 1) {
+					debugger;
+					let i = this.spheres.indexOf(e.target);
+					if (i != -1) {
+						this.removeMarker(i);
+					} else {
+						console.error("Clicked sphere not in list of spheres: ", e);
+					}
+				} else if (window.truthAnnotationMode == 2) {
+					debugger;
```
Comment (bcasselsND) [inline src/utils/Measure.js]: ??

File: src/utils/Measure.js [hunk 4]
```diff
@@ -186,6 +186,33 @@ export class Measure extends THREE.Object3D {
 			sphere.addEventListener('drop', drop);
 			sphere.addEventListener('mouseover', mouseover);
 			sphere.addEventListener('mouseleave', mouseleave);
+			sphere.addEventListener('mouseleave', (e) => {console.log(e);});
+			sphere.addEventListener('mousedown', (e) => {
+				console.log("MOUSEDOWN", e);
+
+				if (window.truthAnnotationMode == 1) {
+					debugger;
```
Comment (bcasselsND) [inline src/utils/Measure.js]: ??

File: data/.~lock.rtk.csv# (hunk 1 above)
Comment (vrajur-nd) [reply data/.~lock.rtk.csv#]: Removed and updated .gitignore

File: demo/radar.html (hunk 2 above)
Comment (vrajur-nd) [reply demo/radar.html]: Removed

File: src/utils/Measure.js (hunk 4 above)
Comment (vrajur-nd) [reply src/utils/Measure.js]: Removed

**Comment by vrajur-nd** [comment]
Pushed a change to address the PR comments - merging this in

**Pull request merged**